### PR TITLE
PXC-2968: Pxc rel 5.7.29

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

**Comment by dutow** [comment]
Rebased on the approved 5.6 commit

File: gcomm/src/gmcast.cpp [hunk 1]
```diff
@@ -12,38 +12,38 @@
...
-using gcomm::gmcast::ProtoMap;
 using gcomm::gmcast::Link;
 using gcomm::gmcast::LinkMap;
 using gcomm::gmcast::Message;
+using gcomm::gmcast::Proto;
+using gcomm::gmcast::ProtoMap;
 
 const long gcomm::GMCast::max_retry_cnt_(std::numeric_limits<int>::max());
 
-static void set_tcp_defaults (gu::URI* uri)
```
Comment (kennt-percona) [inline gcomm/src/gmcast.cpp]: It looks like this file has been reformatted to MySQL style (rather than keeping to galera).

Comment (dutow) [reply gcomm/src/gmcast.cpp]: Fixed.

**Review by kennt-percona — APPROVED** [review]
lgtm

**Pull request merged**